packages/vats/decentral-psm-config.json

```diff
@@ -10,7 +10,9 @@
             "denom": "ibc/usdc1234"
           }
         ],
-        "economicCommitteeAddresses": {}
+        "economicCommitteeAddresses": {
+          "turadgDev": "agoric109q3uc0xt8aavne94rgd6rfeucavrx924e0ztf"
```

Comment:
don't check this in.

Reply:
it's draft. but I still have to figure out how to make the integration tests work without it

Reply:
removed